src/imoh/feature.js

```diff
@@ -0,0 +1,57 @@
+console.log("imoh")
+
+/*
+14 ---------- numbers
+-80 --------- numbers
+8.5 --------- numbers
+true -------- booleans
+(5 < 17) ---- booleans
+null -------- undefined
+3.142 ------- numbers
+"pi" -------- strings
+67+18 ------- function
+*/
+
+
+
+//convert number to base 2
+var base2converter = function(number) {
+	var i = parseInt(number, 10).toString(2);
+	return(i)
+
+}
+base2converter(60);
+
+var score = 180;
+printGrade = function(score) {
+
+	if ( (score <= 100) && (score >= 0)) {
+		if(score >= 70) {
+			return(score+" is A")
```

Comment:
The `return` keyword does not need to be called like a function; you don't need the parentheses.